src/test/java/com/fasterxml/jackson/databind/struct/UnwrappedWithCreatorTest.java

```diff
@@ -0,0 +1,123 @@
+package com.fasterxml.jackson.databind.struct;
+
+import com.fasterxml.jackson.annotation.*;
+import com.fasterxml.jackson.databind.*;
+
+/**
+ * Tests to verify [databind#1467].
+ */
+public class UnwrappedWithCreatorTest extends BaseMapTest
+{
+	static class ExplicitWithoutName {
+		private final String _unrelated;
```

Comment:
@fxshlein I changed internal field names to make 100% sure test could not accidentally work due to field name matching constructor property name (since that would actually work despite `final` field declaration, in absence of Creator parameters).